src/components/TagsPoplist/index.js

```diff
@@ -55,6 +55,7 @@ const TagsPoplist = ({ tags, threshold, maxLength, maxTagWidth, ...props }) => {
       <Box display="flex" alignItems="center" color="gray700" {...props}>
         {tags.slice(0, visibleTagsCount).map((tag, i) => (
           <Tag
+            // eslint-disable-next-line react/no-array-index-key
```

Comment:
Is there a good solution to drop that ?

Reply:
Sadly tag is just a string and we can have twice the same tag so we have to keep the index :(